android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/register/FamilyRegisterActivity.kt

```diff
@@ -21,6 +21,7 @@ import android.os.Bundle
 import android.view.MenuItem
 import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
+import kotlinx.coroutines.runBlocking
 import org.smartregister.fhircore.anc.AncApplication
 import org.smartregister.fhircore.anc.R
 import org.smartregister.fhircore.anc.data.FamilyRepository
```

Comment:
`familyRepository` access modifier can be changed to `private`. Remove unused variable `dispatcherProvider`